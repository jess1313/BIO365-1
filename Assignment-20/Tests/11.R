suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("11.R"))))

print(signTest(c(43, -47, 154, 64, 127, 296, 16, -100, -980, -290, 1090, -8, -78, 70, 20400, -3, 2, 8, 12, 227, 1, 61, 1, 79, 78), 0))

set.seed(33)
print(signTest(rnorm(100), 0))
print(signTest(rnorm(10000), 0))
print(signTest(rnorm(10000, mean=0.1), 0))
print(signTest(rnorm(10000, mean=-0.1), 0))