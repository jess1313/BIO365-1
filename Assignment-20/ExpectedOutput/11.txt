[1] 0.04328525
[1] 0.9204108
[1] 0.6383572
[1] 2.031556e-18
[1] 5.857392e-18
